inst/ms/aggreCAT.qmd

```diff
@@ -664,34 +712,46 @@ and the plausibility ratings under `binary_question` in column
-pre-processes the data to remove missing data, and to return the data
-into an appropriate structure for applying the aggregation function with
-`dplyr::summarise()`.
+`round_2_filter` defaults to `TRUE` (See Table \ref{tbl-method-summary-table} for required inputs of all aggregation methods). `preprocess_judgements()` further pre-processes the data to remove missing data, and to return the data
+into an appropriate structure for calculating weights and applying the
+aggregation function with `dplyr::summarise()`.
 
 ```{r}
 #| label: demo-preprocess-judgements
 #| prompt: true
 data_ratings %>% 
-  group_by(paper_id) %>% 
-  nest()  %>% 
-  ungroup() %>% 
-  sample_n(1) %>% 
-  unnest(cols = c(data)) %>% 
+  dplyr::group_by(paper_id) %>% 
+  tidyr::nest()  %>% 
+  dplyr::ungroup() %>% 
+  dplyr::slice_sample(n = 1) %>% 
+  tidyr::unnest(cols = c(data)) %>% 
   preprocess_judgements()
 ```
 
-After `preprocessing_judgements()` and the aggregation method is
-applied, the function `post_process_judgements()` then processes the
-variables into the final data frame that is returned by each aggregation
-function. The post processing function returns a [tibble]{.class}
-consisting of observations equal to the number of unique claims that
-were parsed to `post_process_judgements()`, the `method`, associated
-`method_id` , `paper_id` , the Confidence Score `cs`, as well as the
-number of participants `n_experts` whose assessments were used in the
-aggregation, and the date of the first and last assessments
-`first_expert_date` and `last_expert_date` respectively.
+The `preprocess_judgements()` function is exposed to the user to allow
+for data formatting in preparation for plotting, e.g. with
+[ggplot2]{.pkg} [@ggplot2016], or for developing bespoke aggregation
+functions / methods not supplied in \code{aggreCAT}.
+
+For some aggregation methods, weights are necessary, and thus are
+computed prior to aggregation. Some aggregation methods compute weights
+using separate weighting functions (See Table \ref{tbl-method-summary-table}), however,
+for aggregation methods with simpler weight computations, these are
+defined in-function, rather than being modularised.
+
+After application of `preprocess_judgements()`, weights are constructed,
+and the aggregation method is applied, the function
+`post_process_judgements()` then processes the variables into the final
+data frame that is returned by each aggregation function. The post
+processing function returns a [tibble]{.class} consisting of
+observations equal to the number of unique claims that were parsed to
+`post_process_judgements()`, the `method`, associated `method_id` ,
+`paper_id` , the Confidence Score `cs`, as well as the number of
+participants `n_experts` whose assessments were used in the aggregation,
+and the date of the first and last assessments `first_expert_date` and
+`last_expert_date` respectively. (David/AAron to check - are we still
```

Comment:
No, `method_id` should have been dropped across the package and incorporated into the pipeline instead since it was only important for DARPA submissions